main.tf

```diff
@@ -77,6 +77,7 @@ module "artifact" {
   filename    = "lambda.zip"
   module_name = "terraform-aws-lambda-elasticsearch-cleanup"
   module_path = "${path.module}"
+  git_ref     = "e7633d0eaef2200bbe0d3dfdd39a87a904c88084"
```

Comment:
@joshmyers 
1. I do not want to lookup correct SHA when going to install that.
2. SHA is kind of version, we pin version of modules, why we can not pin it as SHA?
3. I'm ok to define variable `elasticsearch_cleanup_version` with default value point to the SHA

Reply:
I do not know what curl command was

Reply:
this works without needing to hardcode. We had this working. Something else might have changed, but the strategy works.

Reply:
> Part of the problem of the last approach is that it was very difficult / nay impossible to do local dev as the thing hadn't been build and pushed yet.

It is local dev because we allow all paths to be defined as inputs to the module. So the guy who knows what they are doing can... But it's also possible for someone without nodejs installed locall to deploy this module because it *defaults* to remote artifacts. =)